test/data/event/770fb0b4-0ad8-4774-9275-099b66627355-place-rels.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><metadata xmlns="http://musicbrainz.org/ns/mmd-2.0#"><event type="Concert" id="770fb0b4-0ad8-4774-9275-099b66627355"><name>1987-06-07: Rock am Ring, Nürburgring, Nürburg, Germany</name><life-span><begin>1987-06-07</begin><end>1987-06-07</end></life-span><relation-list target-type="place"><relation type-id="e2c6f697-07dc-38b1-be0b-83d740165532" type="held at"><target>7643f13a-dcda-4db4-8196-3ffcc1b99ab7</target><place id="7643f13a-dcda-4db4-8196-3ffcc1b99ab7"><name>Nürburgring</name><address>Nürburgring Boulevard 1, 53520 Nürburg</address><coordinates><latitude>50.33556</latitude><longitude>6.9475</longitude></coordinates></place></relation></relation-list></event></metadata>
```

Comment:
These files only serve for the mocking of the web request and in such should be identical to what the web request returns (which is without newlines).

Of course, for the actual debugging I look at the result of the request in my browser (where it is automatically formatted).